src/components/Form/FormHelpButton.tsx

```diff
@@ -12,13 +10,21 @@ function safeId(id: string) {
   return id.replace('.', '-')
 }
 
+export interface HelpProps {
+  label: string
+  text: string
```

Comment:
Hi @abrie could `text` be changed to `content` and be `string | React.ReactNode`? When I changed this last, @ivan-aksamentov mentioned that he wanted to allow the content of help possibly be more than plain text.

Reply:
@gj262 Yeah, sounds reasonable. PR Updated.